### PR TITLE
V2.0.1 backports

### PR DESCRIPTION
**Pull request by mheon**
Backport all our non-feature commits to the v2.0 branch, and add release notes for the new v2.0.1 release

### CODE REVIEW

**Comment by mheon** [comment]
@TomSweeneyRedHat PTAL at release notes changes

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
...
+- Fixed a bug where the bash completions did not include the `podman network` command and its subcommands.
+- Fixed a bug where the mount command would not work inside of rootless containers ([#6735](https://github.com/containers/libpod/issues/6735)).
+- Fixed a bug where SSH agent authentication support was not properly working in the `podman-remote` and `podman --remote` commands.
+- Fixed a bug where the `podman untag` command was not erroring when no matching image was found.
+- Fixed a bug where stop signal for containers was not being set properly if not explicitly provided.
+
+### API
+- Fixed a bug where network endpoint URLs in the compatability API were mistakenly suffixed with `/json`.
+- Fixed a bug where the Libpod volume creation endpoint returned 200 instead of 201 on success.
+
```
Comment (TomSweeneyRedHat) [inline RELEASE_NOTES.md:25]: RN changes LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM
assuming happy tests

**Pull request merged**